QRCoder/QRCodeGenerator.cs

```diff
@@ -910,72 +989,81 @@ private static bool IsInRange(char c, char min, char max)
             return (uint)(c - min) <= (uint)(max - min);
         }
 
-        private static Polynom CalculateMessagePolynom(string bitString)
+        private static Polynom CalculateMessagePolynom(BitArray bitArray, int offset, int count)
         {
-            var messagePol = new Polynom();
-            for (var i = bitString.Length / 8 - 1; i >= 0; i--)
+            var messagePol = new Polynom(count / 8);
+            for (var i = count / 8 - 1; i >= 0; i--)
```

Comment:
I'm not sure* if the JIT keeps `count / 8` in a register here (CSE).
So maybe make it explicit and use a separate variable for this?

\* for a simple loop it will, but here it's a `new` and a method call

Reply:
Done